Actions/CalculateArtifactNames/CalculateArtifactNames.ps1

```diff
@@ -17,6 +17,9 @@ $settings = $env:Settings | ConvertFrom-Json
 if ($project -eq ".") {
     $project = $settings.repoName
 }
+elseif ($settings.projectName) {
+    $project = $settings.projectName
```

Comment:
Please don't.

Changing the artifacts name like that is breaking. Just search for `.Replace('\','_').Replace('/','_')` to get an idea of where artifact names could be broken. Not to mention that the `projectName` property is not validated for uniqueness or invalid characters.

 Definitely not part of this effort.
 
 Can `projectName` be utilized directly when generating the doc?

Reply:
I removed it, this just means that docs won't look good in BCApps.
When generating docs for the current branch during CI/CD - yes, this would be possible, but for the releases not, Projects might have changed names or new projects might have been created. I really only have the artifact names. Same goes for the generate docs on a schedule - a change might have been checked in and not yet build that changes things.